snappy/install.go

```diff
@@ -46,10 +46,44 @@ const (
 	AllowOEM
 )
 
-// Update the installed snappy packages, it returns the updated Parts
+func doUpdate(part Part, flags InstallFlags, meter progress.Meter) error {
+	if _, err := part.Install(meter, flags); err == ErrSideLoaded {
+		logger.Noticef("Skipping sideloaded package: %s", part.Name())
+		return nil
+	} else if err != nil {
+		return err
+	}
+	if err := GarbageCollect(part.Name(), flags, meter); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Update updates the selected name
+func Update(name string, flags InstallFlags, meter progress.Meter) ([]Part, error) {
```

Comment:
Yeah, I added the "Installed()" part of the error message. Not sure how relevant this is, but it seemed more correct this way.